src/services/containerServices.ts

```diff
@@ -87,4 +87,21 @@ export class ECSContainer implements IContainerServices {
     }
     return '';
   }
+
+  async stopZombieECSTask(taskId: string) {
+    const clusterName = this._config.get<string>('taskDefinitionFamily');
+    try {
+      await this._client.stopTask({
+        cluster: clusterName,
+        task: taskId,
+      });
+    } catch (error) {
+      // If task not found, then the task should have already been stopped properly
+      if (error.code === 'TaskNotFoundException') {
+        this._logger.error('stopZombieECSTask', error.message);
+      } else {
+        throw error;
+      }
```

Comment:
Need to circle back and double-check the correct exception. Pretty sure both the exception and the syntax are incorrect